sw/device/tests/BUILD

```diff
@@ -1868,3 +1868,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
+    name = "chip_power_idle_load",
+    srcs = ["chip_power_idle_load.c"],
+    deps = [
```

Comment:
Actually, NO.
The test does not need VSEQ to finish, it is really self contained.
VSEQ was added due to request from other PR reviewers...
The S/W activates AON timer. When timer expires, it issues NMI and the S/W should resume.
I guess, the root cause is that FPGA does not allow for sufficient run time until AON timer expires.
I don't have access to running FPGA simulations so I can not check this further,
But we should either increase FPGA test timeout or
shorten AON timer timeout in case FPGA platform is used.

Reply:
I tried both things but the test seems to hang. I will take a closer look.

Reply:
I replaced `wait_for_interrupt()` for `IBEX_SPIN_FOR(nmi_fired, kTimeTillBarkUsec * 2);` to get the FPGA test to pass.  

Just to confirm, does this test require `wfi`?

Reply:
Switched to `IBEX_SPIN_FOR` to re-run CI.

Reply:
This test is intended to check the Chip load (power consumption) while the CPU is in idle mode.
So I think the "wfi" instruction is a must here.